netty/src/main/java/io/grpc/netty/CreateStreamCommand.java

```diff
@@ -27,16 +28,21 @@
   private final Http2Headers headers;
   private final NettyClientStream.TransportState stream;
   private final boolean get;
+  private final CallOptions callOptions;
 
-  CreateStreamCommand(Http2Headers headers,
-                      NettyClientStream.TransportState stream) {
-    this(headers, stream, false);
+  CreateStreamCommand(
+      Http2Headers headers,
+      NettyClientStream.TransportState stream) {
+    this(headers, stream, CallOptions.DEFAULT, false);
```

Comment:
Lying about the CallOptions worries me. Marking this a Deprecated or VisibleForTesting would make me feel better.